src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -511,7 +516,7 @@ export default {
 }
```

Comment:
The line right above here should be `@apply tw-text-primary tw-no-underline;`, otherwise the loan use anchor gets underlined on focus.

Reply:
Fixed